config/routes.rb

```diff
@@ -2,5 +2,8 @@
   get '/health', to: 'health#show'
   get '/robots.txt', to: 'robots_txts#show'
 
+  get '/session/remaining', to: 'session#remaining'
+  get 'session/extend', to: 'session#extend'
```

Comment:
Should this be a POST request (protected by CSRF)?  I feel like *technically* it sholuld, as it is changing app state.  Is there any security/threat here?

I suppose it means anyone can ping this url and theoretically extend someone elses session... nut is that likely...?

Reply:
It seems unlikely though bots will find and ping the route. What happens if you `curl` the route? I imagine browserless `remaining` will return nothing? `extend` will do nothing?